content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/file-and-directory-naming/general
+ready: true
+tags:
+- File and directory naming
+- python
+title: File and directory naming in Python
+---
+
+## How should I name my Python modules and packages?
+
+You should name your files and directories the same way you name Python variables.They should be small letters, and words should be separated by underscores, as that improves readability.
+
+It is important to note that the use of underscores in directory_naming takes preference as opposed to the use of dashes.
```

Comment:
Would you ever talk like this in real life? 

How can you say this more simply? 

KISS applies to words as well.

You can also use examples to show what you mean. eg:

```
good_directory_name
bad-directory-name
AnotherBadDirectoryName
```